tests/lca/imagebasedupgrade/cnf/internal/validations/post_ibu_validations.go

```diff
@@ -39,64 +46,355 @@ func PostUpgradeValidations() {
-					Expect(cnfclusterinfo.PreUpgradeClusterInfo.ID).
-						To(Equal(cnfclusterinfo.PostUpgradeClusterInfo.ID), "Cluster ID has changed")
-				})
-			})
+			Expect(recFound).To(BeTrue(), "Could not find any of the node hostname DNS records in the network configuration")
+		})
+	})
+}
 
-			It("Validate no pods using seed name", reportxml.ID("71389"), Label("ValidatePodsSeedName"), func() {
-				By("Validate no pods are using seed's name", func() {
-					podList, err := pod.ListInAllNamespaces(TargetSNOAPIClient, v1.ListOptions{})
-					Expect(err).ToNot(HaveOccurred(), "Failed to list pods")
+// ValidateSeedHostnameRefLogs check if no seed hostname references are present in pod logs.
+func ValidateSeedHostnameRefLogs() {
+	It("Validate no seed hostname references in pod logs", reportxml.ID("71392"), Label("ValidateSeedRefPodLogs"), func() {
+		By("Validate no seed hostname references in pod logs", func() {
+			logCmd := "grep -Ri " + seedInfo.SNOHostname + " /var/log/pods | grep -v lifecycle-agent-controller-manager | wc -l"
+			logRes, err := cluster.ExecCmdWithStdout(TargetSNOAPIClient, logCmd)
+			Expect(err).ToNot(HaveOccurred(), "could not execute command: %s", err)
+
+			for _, stdout := range logRes {
+				seedLogRef, err := strconv.Atoi(strings.ReplaceAll(stdout, "\n", ""))
+				Expect(err).ToNot(HaveOccurred(), "could not convert string to int: %s", err)
+				Expect(seedLogRef).ToNot(BeNumerically(">", 0), "Seed hostname references detected in pod logs: %s", stdout)
+			}
+		})
+	})
+}
 
-					for _, pod := range podList {
-						Expect(pod.Object.Name).ToNot(ContainSubstring(seedInfo.SNOHostname),
-							"Pod %s is using seed's name", pod.Object.Name)
+// ValidateSeedHostnameRefEtcd check if no seed hostname references are present in etcd.
+func ValidateSeedHostnameRefEtcd() {
+	It("Validate no seed hostname references in etcd", reportxml.ID("71393"), Label("ValidateSeedRefEtcd"), func() {
+		By("Validate no seed hostname references in ectd", func() {
+			etcdPod, err := pod.ListByNamePattern(TargetSNOAPIClient, "etcd", "openshift-etcd")
+			Expect(err).ToNot(HaveOccurred(), "Failed to get etcd pod")
+
+			etcdCmd := fmt.Sprintf(
+				"for key in $(etcdctl get --prefix / --keys-only | grep -v ^$); do"+
+					"  value=$(etcdctl get --print-value-only $key); "+
+					"  if [[ $value == *%s* ]]; then"+
+					"    echo Key: $key contains seed reference;"+
+					"    break; "+
+					"  fi;"+
+					"done",
+				seedInfo.ClusterName,
+			)
+
+			getKeyCmd := []string{"bash", "-c", etcdCmd}
+			getKeyOut, err := etcdPod[0].ExecCommand(getKeyCmd, "etcdctl")
+			Expect(err).ToNot(HaveOccurred(), "could not execute command: %s", err)
+			Expect(getKeyOut.String()).ToNot(
+				ContainSubstring("contains seed reference"),
+				"Seed hostname references detected in etcd: %s",
+				getKeyOut.String(),
+			)
+		})
+	})
+}
+
+// ValidateDUConfig check if pre-upgrade DU configs are present after upgrade.
```

Comment:
In my view, PTP configuration check would have been part of  "ValidateDUConfig" function and it would be nice to have this check after the upgrade. what do you think? 

`Step:	
Check specific configurations such as SriovNetworkNodePolicies, SriovNetworks, PTPConfig, clusterID, etc remain the same as before upgrade    Expected Result:	The configurations get preserved post-upgrade`

Reply:
I agree but the ptp pkg is currently empty in eco-goinfra so I'll have to extend it after adding the required functions to the ptp pkg.

Reply:
In this case, I would prefer go ahead with your approach i.e. keeping sriovnetworks and sriovnetworkpolicies configuration as the DU config check after the upgrade at this moment. we can add ptp configuration check later during the course of action.